lib/serializables/log-event.js

```diff
@@ -32,7 +32,18 @@ function LogEventFactory (
     pretty,
     console
 ) {
+    //default turn off the colorful output to achieve tidy and clean log file.
+    //however user can still enable it via command line argument.
+    var colorEnable = false;
+    var argv = process.argv;
```

Comment:
I planed to use the configuration service to define the log color option, but stopped it because let the LogEvent depends on the Configuration Service is not good design, so that's my compromised design by only examining the process.env.

Since nfcon already process the command line options, so we don't need the "commander" though it is a very powerful library even only look at its star number :). Move to index.js will not have the dependency concern.

Our design of Configuration Service should be a combination of handling arguments from command line options, system environment and config file.

So I will leverage the Configuration Service to define the log color switch, so user can either enable  the log color from command line option or from monorail.json.